project-api/ballerina-projects/src/test/java/io/ballerina/projects/test/TestBuildProject.java

```diff
@@ -199,4 +209,334 @@ public void testSetBuildOptionsFromToml() {
         Assert.assertTrue(project.getBuildOptions().isSkipLock());
     }
 
+    @Test
+    public void testUpdateDocument() {
+        // Inputs from langserver
+        Path filePath = RESOURCE_DIRECTORY.resolve("myproject").resolve("main.bal").toAbsolutePath();
+        String newContent = "import ballerina/io;\n";
```

Comment:
Can we change this to a simple constant importing ballerina/io will not compile since the stdlibs are removed.

Reply:
Changed in 918ddd9